onnxruntime/test/testdata/custom_op_library/custom_op_library.cc

```diff
@@ -6,9 +6,31 @@
 
 #include <vector>
 #include <cmath>
+#include <mutex>
 
 static const char* c_OpDomain = "test.customop";
 
+struct OrtCustomOpDomainDeleter {
+  OrtCustomOpDomainDeleter(const OrtApi* ort_api) {
+    ort_api_ = ort_api;
+  }
+  void operator()(OrtCustomOpDomain* domain) const {
+    ort_api_->ReleaseCustomOpDomain(domain);
```

Comment:
>ReleaseCustomOpDomain [](start = 14, length = 21)

This needs better documentation. Does this release another library?

Reply:
No. This just calls an exposed ORT API to release custom op domain (This is part of the API struct)